### PR TITLE
Stt text to ground truth outlier remover

### PR DESCRIPTION
**Pull request by loganhart02**
Some things to point out. when building the docker image the .opus file fails because stt cant transcribe those files. we also need to add add the huge tflite model and huge vocab scorer to the repo. It won't let me push files that big

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,13 @@
 FROM ghcr.io/coqui-ai/stt-train:main
 
 RUN python -m pip install pandarallel
+RUN python -m pip install stt
+RUN python -m pip install numpy
+RUN python -m pip install librosa
 
 WORKDIR /home/ubuntu
 COPY . /home/ubuntu/
```
Comment (JRMeyer) [inline Dockerfile:10]: RUN mkdir /home/ubuntu/stt_model
RUN wget https://github.com/coqui-ai/STT-models/releases/download/english%2Fcoqui%2Fv1.0.0-huge-vocab/model.tflite -P /home/ubuntu/stt_model
RUN wget https://github.com/coqui-ai/STT-models/releases/download/english%2Fcoqui%2Fv1.0.0-huge-vocab/huge-vocabulary.scorer -P /home/ubuntu/stt_model

Comment (JRMeyer) [inline Dockerfile:10]:
```suggestion
RUN mkdir /home/ubuntu/stt_model
RUN wget https://github.com/coqui-ai/STT-models/releases/download/english%2Fcoqui%2Fv1.0.0-huge-vocab/model.tflite -P /home/ubuntu/stt_model
RUN wget https://github.com/coqui-ai/STT-models/releases/download/english%2Fcoqui%2Fv1.0.0-huge-vocab/huge-vocabulary.scorer -P /home/ubuntu/stt_model
```

Comment (JRMeyer) [reply Dockerfile:10]: @loganhart420 let's grab the acoustic and language model via the dockerfile

**Comment by JRMeyer** [comment]
> when building the docker image the .opus file fails because stt cant transcribe those files. 

in this case, running `stt` should be optional

File: Dockerfile (hunk 1 above)
Comment (loganhart420) [reply Dockerfile]: Thanks I wasn't sure how to handle it

**Comment by loganhart420** [comment]
> > when building the docker image the .opus file fails because stt cant transcribe those files.
> 
> in this case, running `stt` should be optional

I used an if statement to check the length of sys.argv and if its more than 3 it enables it since the last two args would be the stt model and scorer. Is that an okay way to do it?

**Pull request merged**